test/integration-ebpf/src/stack_argument.rs

```diff
@@ -0,0 +1,70 @@
+#![no_std]
+#![no_main]
+
+use aya_bpf::{
+    macros::{map, uprobe},
+    maps::PerfEventArray,
+    programs::ProbeContext,
+};
+use aya_log_ebpf::{debug, info};
+
+pub struct Args {
+    a_0: u64,
+    a_1: u64,
+    a_2: u64,
+    a_3: u64,
+    a_4: u64,
+    a_5: u64,
+
```

Comment:
why this space? perhaps a better convention is to name these arg{0..5} and stack_arg{0..5}